data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+			`	You then follow the Unfettered captain to the hatch. He is well-prepared and quite deft, even given that he is wearing what seems to be a heavy combat suit.`
+			choice
+				`	(No time for talking.)`
+					goto outsideship
+				`	"Who made this vault? What should I expect?"`
+				`	"Shouldn't we take more men to ensure our safety?"`
+					goto moremen
+			`	"Arrr, I cannot be telling anyone that, not even ye after bringing me, and especially not here. I dare not risk bringing danger to their door or their ire to mine. At least while we still need their supply of jump drives. For now, just think of them as more pirates, like me and the rest of the scallywags on this bilge water planet here, and know that we be ready to face them in battle, should it come to that," he tells you.`
+				goto outsideship
+			label moremen
+			action
+				set "unfettered: dumb Si-Ra question"
+			`	"We be needing discretion here, not to draw attention! The larger our group is, the more suspicion we attract. The more suspicion we attract, the more likely someone fits us for a hempen halter. We be wanting them to think we hold no loot, not that we be hunting treasure, savvy?" Judging from his tone, this was a stupid question.`
+			label outsideship
+			`	Before exiting, he warns you, "Whatever be happening, do not try anything stupid and stay behind me at all times."`
+			`	Once outside, he moves at a quick pace, and you have difficulties following him, especially with the terrain as you head into the mountains. He seems worried and keeps looking around furtively. All things considered, he is impressively stealthy; the suit must have been crafted specifically for him. He presses a switch every so often which pops out a mirror, allowing him to peer cautiously around corners and obstacles in your path in case of an ambush.`
+			`	Before long, you arrive at a small shanty town constructed with metal and other scraps the builders could gather nearby. Even considering what it is, it appears recently constructed and especially ramshackle.`
+			choice
+				`	(Keep walking, ready to reach for my gun.)`
+					goto discreet
+				`	"Is everything alright?"`
+				`	"There is no need to be worried, that cache must be well hidden, right?"`
+			action
+				"unfettered rep: Si-Ra" --
+			`	He makes some kind of sign you do not understand, but it seems pretty clear he wants you to keep quiet. Looking around, you notice the two of you have attracted quite a bit of attention from the impoverished locals. While they all seem to have nothing to lose, the vast majority of them are very young and probably not the hardened criminals you had expected.`
+			label discreet
+			action
+				"unfettered rep: Si-Ra" ++
```

Comment:
So this cancels out the loss you just had earlier?

Reply:
yeah I need to take a closer look at this haha